boefjes/boefjes/katalogus/types.py

```diff
@@ -14,6 +15,7 @@ class PaginationParameters(BaseModel):
 
 class FilterParameters(BaseModel):
     q: str | None = None
+    ids: list[str] | None = Query(None)
```

Comment:
I don't think assigning this to `Query` is appropriate:
- The type is incorrect; you'd expect a `list` or `None` type
- `Query` is a special field in FastAPI used for injecting query parameters dependencies from requests, Pedantic doesn't know anything about this

```suggestion
    ids: list[str] | None = None
```